cmd/docs/README.md

```diff
@@ -0,0 +1,97 @@
+# WSO2 Micro Integrator Command Line tool (MI-CLI)
+
+View and manage the micro integrator runtime using the “mi” command line tool. Some of the usages of the command line tool include,
+1. Get a list of deployment runtime artifacts
+2. Inspect details of each runtime artifact such as a proxy service or an API.
+3. Get the invocation endpoint of an artifact
+
+### Running
+
+Extract the compressed archive generated to the desired location.
```

Comment:
```suggestion
1. Extract the compressed archive generated to the desired location.
```

Reply:
Since we pack the README inside the archive, this step was removed in cdbd7df